jlm/llvm/backend/jlm2llvm/jlm2llvm.cpp

```diff
@@ -264,7 +264,8 @@ convert_attribute_kind(const attribute::kind & kind)
         { attribute::kind::VScaleRange, ak::VScaleRange },
         { attribute::kind::LastIntAttr, ak::LastIntAttr },
 
-        { attribute::kind::EndAttrKinds, ak::EndAttrKinds } });
+        { attribute::kind::EndAttrKinds, ak::EndAttrKinds },
+        { attribute::kind::NoFPClass, ak::NoFPClass } });
```

Comment:
The NoFPClass attribute should not go last I believe. You see that all these attribute blocks have First/Last pairs. Check where it is located in the the LLVM attribute list

Reply:
Moved